python/lsst/obs/base/cameraMapper.py

```diff
@@ -268,6 +272,12 @@ def __init__(self, policy, repositoryDir,
 
         self.makeRawVisitInfo = self.MakeRawVisitInfoClass(log=self.log)
 
+        # Assign a metadata translator if one has not been defined by
+        # subclass. We can sometimes infer one from the RawVisitInfo
+        # class
```

Comment:
I fee like this attempt to infer functionality should be documented clearly in the class docstring. Also, last sentence missing a period.

Related to the class docstring, I just noticed that `__init__` has the parameters docstring, not the class: please fix that.